cmds/identityd/main.go

```diff
@@ -1,24 +1,33 @@
 package main
 
 import (
+	"context"
 	"flag"
 	"os"
 
+	"github.com/threefoldtech/zbus"
+
 	"github.com/cenkalti/backoff"
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 	"github.com/threefoldtech/zosv2/modules/identity"
 	"github.com/threefoldtech/zosv2/modules/version"
 )
 
-const seedPath = "/var/cache/seed.txt"
+const (
+	seedPath = "/var/cache/seed.txt"
```

Comment:
Let's create a dedicated directory on the cache for identityd and store all its files in there instead